### ISSUE TITLE
InitialCameraUpdate parsing

### ISSUE THREAD

**Issue by CodingContraption** [amay077/Xamarin.Forms.GoogleMaps#421]
### SUMMARY

When supplying InitialCameraUpdate in XAML on certain cultures that use commas instead of dots for decimals ( so far only tested on Dutch devices), the values are parsed incorrectly. I suspect the parsing of the string is not invariant to cultures.

### VERSIONS

* Xamarin.Forms.GoogleMaps - 2.3.0-beta1
* Xamarin.Forms - 2.5.0.91635

### PLATFORMS

- [x] Android
- [x] iOS
- [ ] UWP (not tested)

### HOW TO REPRODUCE

1. Initialise a map like below
2. Run on a device set to a culture that uses commas instead of dots for double strings (In my use case: dutch, I believe canadian also uses commas.).
3. Notice the map is not on the expected location (it should show the netherlands)

```csharp
<maps:Map x:Name="map"
            InitialCameraUpdate="52.2,5.3,7"
            VerticalOptions="FillAndExpand"
            HorizontalOptions="FillAndExpand"
            Grid.Row="0"
            Grid.Column="0" />
```

**Comment by CodingContraption** [comment]
I have created a pull request that should fix this issue: [#422](https://github.com/amay077/Xamarin.Forms.GoogleMaps/pull/422)

**Comment by amay077** [comment]
Thanks for your great PR! 🥇 
I'll merge it soon.

**Comment by amay077** [comment]
I merged your PR and released [v2.3.0-beta2](https://www.nuget.org/packages/Xamarin.Forms.GoogleMaps/2.3.0-beta2) .
You can try it now.

**Comment by CodingContraption** [comment]
I have tested it on multiple cultures and can confirm that the issue is solved. Thanks for the quick merge! 👍
